packages/payload/src/fields/validations.ts

```diff
@@ -470,6 +470,7 @@ export const blocks: Validate<unknown, unknown, BlockField> = (
   value,
   { maxRows, minRows, required, t },
 ) => {
+  const hasValue = value && Array.isArray(value) && value.length > 0
```

Comment:
I think we can remove this and just add an if statement before the min/max rows check

```ts
if (!required && !arrayLength) return true
```